components/DraggableItem/DraggableItem.tsx

```diff
@@ -0,0 +1,67 @@
+import { FC, PropsWithChildren } from 'react';
+import { Image } from '../Image';
+import { IconButton } from '../IconButton';
+import { LinkIcon, PencilIcon, TrashIcon } from '@heroicons/react/solid';
+
+interface Props {
+  text: string;
+  icon?: string;
+  url?: string;
+}
+
+export const DraggableItem: FC<Props> = ({ text, icon, url }) => {
+  const Wrapper = url
+    ? ({ children, ...props }: PropsWithChildren) => (
+        <a
+          href={url}
+          {...props}
+        >
+          {children}
+        </a>
+      )
+    : 'span';
```

Comment:
it's better if u move this component outside `DraggableItem`

Reply:
Updated at [b494200](https://github.com/chalalala/remo/pull/9/commits/b49420008e1bf2b55bed619ffc8bf65ec783d3eb)